### DISCUSSION TITLE
Default audio devices + saving audio device per player

### DISCUSSION THREAD

**Discussion by nick-potts** [f1multiviewer/issue-tracker#114] [Feature requests] [answered]
I'd love a setting for a default audio device. I'd also like to be able to save a layout and it keep the selected audio devices per player. Also being able to switch stream and it keep the same audio player would be nice.

**Comment by f1multiviewer** [comment, marked as answer]
It'll save it in the layout already, so if you select them, save it, given the audio device IDs don't change, it'll set them the next time you open it!

**Comment by JJWatMyself** [comment]
I think default audio is a different feature so saving with layout.

I agree on better default audio track selection would be great. But also to note that this is mainly because FX being used 'by default' in some situations instead of commentary audio track, e.g. English.  For OBC, team radio would generally be the preferred audio, etc. So would not want to fix one issue and cause another.
